app/controllers/users.js

```diff
@@ -2,16 +2,30 @@ const usersServices = require('../services/users');
 const { userSerializer } = require('../serializers/users');
 
 const {
+  authMessages: { LOGGED },
+  statusCode: { CREATED_CODE, OK_CODE },
   statusMessages: { CREATED }
 } = require('../constants');
 
 exports.createUser = (req, res, next) =>
   usersServices
     .createUser(req.body)
     .then(userInfo =>
-      res.status(201).send({
+      res.status(CREATED_CODE).send({
         user: userSerializer(userInfo),
         message: CREATED
       })
     )
     .catch(next);
+
+exports.signIn = async ({ body: { email, password } }, res, next) => {
+  await usersServices
```

Comment:
Please do not mix async/await and promises, use only one of the two options

Reply:
Totally. You are right. Sorry for this "layer 8" jaja. Solved!